### PR TITLE
set matrix dimensions in explanation identical to code

### PR DESCRIPTION
**Pull request by Reddine**

### CODE REVIEW

File: Minimal ESN - EN.ipynb [hunk 1]
```diff
@@ -135,7 +135,7 @@
     "\n",
     "<p style=\"text-align:justify\">Input neurons are connected to reservoir neurons. We save the weights of every connection in a <b style=\"color:#99cc00\">W<sub>in</sub></b> matrix (dim. inSize x resSize). Weights of the connections from reservoir to the output are saved in the <b style=\"color:#ff0000\">W<sub>out</sub></b> matrix (dim. resSize x outSize). Into the reservoir itself, one neuron is linked to every other : weights are saved in the <b style=\"color:#ffcc00\">W</b> matrix (dim. resSize x resSize).</p>\n",
     "\n",
-    "<p style=\"text-align:justify\">Values of the reservoir's neurons are saved through time in a <b style=\"color:#ffcc00\">x</b> matrix (dim. resSize × (initLen+trainLen)). <b style=\"color:#00baff\">X</b> matrix contains bias (<b style=\"color:#99cc00\">1</b>), input (<b style=\"color:#99cc00\">u</b>) and reservoir's neurons (<b style=\"color:#ffcc00\">x</b>) values, over time (t). This matrix has a (1+inSize+resSize)×(initLen+trainLen) dimension. Finally, we save the ouput neuron values over time in a <b style=\"color:#ff0000\">y</b> matrix (dim. 1×(initLen+trainLen)).</p>\n",
```
Comment (neuronalX) [inline Minimal ESN - EN.ipynb:138]: why did you remove "through time" in the sentence?

**Comment by Reddine** [comment]
Because I saw in the code, "x" hold the value of neurons just at time "t"
not through time.

**Review by neuronalX — APPROVED** [review]

**Comment by neuronalX** [comment]
OK for the changes in the text.
